apix-docker/232/docker-compose.yml

```diff
@@ -15,15 +16,15 @@ services:
       - GLOBAL_localTransform.core-aio.url=http://transform-core-aio:8090/
 
   solr:
-    image: private.docker.xenit.eu/alfresco-enterprise/alfresco-solr6:2.0.3
+    image: private.docker.xenit.eu/alfresco-enterprise/alfresco-solr6:2.0.8.2 # TODO: needs upgrade?! Search: alfresco-search-services-2.0.9.1.zip https://hub.alfresco.com/t5/alfresco-content-services-blog/alfresco-community-edition-23-2-release-notes/ba-p/318918
```

Comment:
Following the documentation we need solr 2.0.9.1. This is not yet build and published by us. 
Seems okay with 2.0.8.2 for testing

Reply:
creating ticket to create new image in other project
ticket:https://xenitsupport.jira.com/browse/DOCKER-456

Reply:
Finished pushing new Version and importing it in docker setup.